frontend/public/components/operator-lifecycle-manager/descriptors/status/index.tsx

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable no-undef, no-unused-vars */
+
+import * as React from 'react';
+import * as _ from 'lodash-es';
+import { Map as ImmutableMap } from 'immutable';
+
+import { ResourceLink } from '../../../utils';
+import { StatusCapability, CapabilityProps, DescriptorProps } from '../types';
+import { Phase } from './phase';
+import { PodStatusChart } from './pods';
+
+// TODO(alecmerdler): Move these to separate module?
+const Default: React.SFC<StatusCapabilityProps> = ({value}) => _.isEmpty(value) && !_.isNumber(value)
```

Comment:
We should add a block comment before each of these describing how to instantiate (its full name) and what it does